src/modules/fancyzones/editor/FancyZonesEditor/CanvasEditorWindow.xaml

```diff
@@ -34,7 +34,7 @@
         <StackPanel Margin="16"
                     FocusManager.FocusedElement="{Binding ElementName=newZoneButton}">
 
-            <Button x:Name="newZoneButton"
+            <local:ClickAutomationEventButton x:Name="newZoneButton"
                     AutomationProperties.LabeledBy="{Binding ElementName=newZoneName}"
```

Comment:
Since you're touching this code, could you please remove this line? There is no `newZoneName` element so it causes an error:

`System.Windows.Data Error: 4 : Cannot find source for binding with reference 'ElementName=newZoneName'. BindingExpression:(no path); DataItem=null; target element is 'ClickAutomationEventButton' (Name='newZoneButton'); target property is 'LabeledBy' (type 'UIElement')`

Reply:
Done